README.md

```diff
@@ -0,0 +1,24 @@
+# Py Unittest Walkthrough
+
+This repository is designed to be a learning tool for learning the usage and best practices of the Python unittest package 
+
+
+## Usage 
+
+To see how it works, if you have python3x installed you can run individual tests by:
+
+    cd test
+    cd unit_tests
+    python3 utils_test.py
```

Comment:
There's a prerequisite to this step for those getting started. 
```
❯ python3 utils_test.py
Traceback (most recent call last):
  File "utils_test.py", line 4, in <module>
    import funprog.utils as utils
ModuleNotFoundError: No module named 'funprog'
```
Since this is a repository for folks getting started with Python testing. I suggest having a `## Prerequisites` section above `## Usage` noting to have python 3.x installed (or link to https://www.python.org/downloads/) and then also link to some information on how to get the dependencies installed.

Reply:
That could be useful, the specific pre-req here is adding the `bin` dir to your `PATH` and the `src` dir to your `PYTHONPATH`. I can potentially add this in case @xanderyzwich needs some help.

Reply:
@drkennetz is that the best direction to require people to make path changes in order to run this?

Reply:
@xanderyzwich as discussed in slack chat, this may be better handled with a script that people source whenever they are inside a directory:
```bash
export PATH=$(pwd)/bin:$PATH
export PYTHONPATH=$(pwd)/src:$PYTHONPATH
```